xzre.h

```diff
@@ -867,11 +876,7 @@ typedef struct __attribute__((packed)) {
 
 	elf_handles_t elf_handles;
 
-	/**
-	 * @brief points to the beginning of this struct
-	 */
-	struct backdoor_data *backdoor_data;
-	elf_handles_t * elf_handles_ptr;
+	backdoor_data_handle_t data_handle;
```

Comment:
this struct is passed as a pointer to find_dl_audit_offsets and the code only makes sense if these two fields are actually a nested struct